images/win/scripts/Installers/Install-Chrome.ps1

```diff
@@ -2,56 +2,8 @@
 ##  File:  Install-Chrome.ps1
 ##  Desc:  Install Google Chrome
 ################################################################################
-function Stop-SvcWithErrHandling
-{
-    param (
-        [Parameter(Mandatory, ValueFromPipeLine = $true)] [string] $ServiceName
-    )
 
-    Process {
-        $Service = Get-Service $ServiceName -ErrorAction SilentlyContinue
-        if (-not $Service) {
-            Write-Error "[!] Service [$ServiceName] is not found";
-            exit 1;
-        }
-        else {
-            Write-Host "Try to stop service [$ServiceName]";
-            try {
-                Stop-Service -Name $ServiceName -Force;
-                $Service.WaitForStatus("Stopped", "00:01:00");
-                Write-Host "Service [$ServiceName] has been stopped successfuly";
-            }
-            catch {
-                Write-Error "[!] Failed to stop service [$ServiceName] with error:"
-                $_ | Out-String | Write-Error;
-            }
-        }
-    }
-}
-
-function Set-SvcWithErrHandling 
-{
-    param (
-        [Parameter(Mandatory, ValueFromPipeLine = $true)] [string] $ServiceName,
-        [Parameter(Mandatory)] [hashtable] $Arguments
-    )
-
-    Process {
-        $Service = Get-Service $ServiceName -ErrorAction SilentlyContinue
-        if (-not $Service) {
-            Write-Warning "[!] Service [$ServiceName] is not found";
-        }
-        try {
-           Set-Service $ServiceName @Arguments;
-        }
-        catch {
-            Write-Error "[!] Failed to set service [$ServiceName] arguments with error:"
-            $_ | Out-String | Write-Error;
-        }
-    }
-}
-
-Import-Module -Name ImageHelpers -Force;
+Import-Module -Name "D:\GitHub\Akvelon\Forks\virtual-environments\images\win\scripts\ImageHelpers\ImageHelpers.psm1" -Force;
```

Comment:
This is absolute path on your local machine. Please use relative path to the module

Reply:
Fixed
Thanks